docs/avifenc.1.md

```diff
@@ -0,0 +1,235 @@
+
+**-s**, **\--speed** _S_
+:   Encoder speed.
+    Default is 6.
+
+    Possible values:
+
+    :   - **0**-**10** (slowest-fastest)
+        - **default**, **d** (codec internal defaults)
+
+**-c**, **\--codec** _C_
+:   AV1 codec to use.
+    Possible values depend on the codecs enabled at build time (see **\--help**
+    or **\--version** for the available codecs).
+    Default is auto-selected from the available codecs.
+
+    Possible values:
+
+    :   - **aom**
+        - **rav1e**
+        - **svt**
+
+**\--exif** _FILENAME_
+:   Provide an Exif metadata payload to be associated with the primary item.
+
+**\--xmp** _FILENAME_
+:   Provide an XMP metadata payload to be associated with the primary item.
+
+**\--icc** _FILENAME_
+:   Provide an ICC profile payload to be associated with the primary item.
+
+**-a**, **\--advanced** _KEY_[_=VALUE_]
+:   Pass an advanced, codec-specific key/value string pair directly to the
+    codec.
+    **avifenc** will warn on any not used by the codec.
+    The aom-specific advanced options can be used if the AOM codec is available
+    (see **\--help** for details).
+
+**\--duration** _D_
+:   Set all following frame durations (in timescales) to _D_.
+    Can be set multiple times (before supplying each filename).
+    Default is 1.
+
+**\--timescale**, **\--fps** _V_
+:   Set the timescale to _V_.
+    If all frames are 1 timescale in length, this is equivalent to frames per
+    second.
+    If neither duration nor timescale are set, **avifenc** will attempt to use
+    the framerate stored in a y4m header, if present.
+    Default is 30.
+
+**-k**, **\--keyframe** _INTERVAL_
+:   Set the forced keyframe interval (maximum frames between keyframes).
+    Set to **0** to disable.
+    Default is 0.
+
+**\--ignore-icc**
+:   If the input file contains an embedded ICC profile, ignore it (no-op if
+    absent).
+
+**\--pasp** *H***,***V*
+:   Add pasp property (aspect ratio).
+
+    - _H_ = horizontal spacing
+    - _V_ = vertical spacing
+
+**\--crop** *CROPX***,***CROPY***,***CROPW***,***CROPH*
+:   Add clap property (clean aperture), but calculated from a crop rectangle.
+
+    - _CROPX_ = X-axis of a crop rectangle
+    - _CROPY_ = Y-axis of a crop rectangle
+    - _CROPW_ = width of a crop rectangle
+    - _CROPH_ = height of a crop rectangle
+
+**\--clap** *WN***,***WD***,***HN***,***HD***,***HON***,***HOD***,***VON***,***VOD*
+:   Add clap property (clean aperture).
+
+    - _WN_ = numerator of width
+    - _WD_ = denominator of width
+    - _HN_ = numerator of height
+    - _HD_ = denominator of height
+    - _HON_ = numerator of horizontal offset
+    - _HOD_ = denominator of horizontal offset
+    - _VON_ = numerator of vertical offset
+    - _VOD_ = denominator of vertical offset
+
+**\--irot** _ANGLE_
+:   Add irot property (rotation).
+    Possible values are **0**-**3**, and makes (90 * _ANGLE_) degree rotation
+    anti-clockwise.
+
+**\--imir** _MODE_
+:   Add imir property (mirroring).
```

Comment:
This was not a suggestion to insert verbatim. I was wondering if the imir property was applied before or after the irot property when rendering a decoded image, and thought it would be useful to write it down here.

MIAF - ISO 23000-22 - Section 7.3.6.7 says

> These properties, if used, shall be indicated to be applied in the following order: clean aperture first, then
rotation, then mirror

So I suggest replacing "It might be useful to specify if it is applied before or after the rotation." by the following:

> Note: Rotation is applied before mirroring at rendering.

Reply:
Fixed this in 92c42b5.